tests/specialization_constants/specialization_constants_exceptions_catching_common.h

```diff
@@ -0,0 +1,82 @@
+/*******************************************************************************
+//
+//  SYCL 2020 Conformance Test Suite
+//
+//  Common expected exceptions catching for specialization constants
+//
+*******************************************************************************/
+
+#ifndef __SYCLCTS_TESTS_SPEC_CONST_EXCEPT_CATCH_COMMON_H
+#define __SYCLCTS_TESTS_SPEC_CONST_EXCEPT_CATCH_COMMON_H
+
+#include "specialization_constants_common.h"
+
+template <typename T>
+class dummy_specialization_constants_exceptions {};
+template <typename T>
+using spec_const_except_dummy_functor =
+    ::dummy_functor<dummy_specialization_constants_exceptions<T>>;
+
+template <typename T>
+class check_spec_constant_except_catch_for_type {
+ public:
+  void operator()(sycl_cts::util::logger &log, const std::string &type_name) {
+    using namespace get_spec_const;
+
+    // case 1: Try to get specialization constant via handler that bound to a
+    // kernel_bundle
+    {
+      T res = T(value_helper<T>(0));
+      const int case_num = 1;
+      auto queue = sycl_cts::util::get_cts_object::queue();
+
+      queue.submit([&](sycl::handler &cgh) {
+        auto context = queue.get_context();
+        auto k_bundle =
+            sycl::get_kernel_bundle<sycl::bundle_state::executable>(context);
+        cgh.use_kernel_bundle(k_bundle);
+        try {
+          res = cgh.get_specialization_constant<spec_const<T, case_num>>();
+        } catch (const sycl::exception &e) {
```

Comment:
It is not super clear what you are trying to do without any comment explaining the intent.
Are you testing that you really generate a given exception?
If so, what if there is no exception at all?

Reply:
i've added comment about this in this commit: ba22e08858bf498a0449b391dd9681522ffa4a85
i hope that this my clarifications can help describe that happens in this tests

Reply:
@kenrussell Sorry, I will answer your question soon

Reply:
@keryell Ronan I've added comments about what exceptions we expect and add statements that verifies exception was thrown otherwise test will failed
I've do in this commit: ce8ec733b01eb9a186189756555731566ed5667a